background/hardware.tex

```diff
@@ -136,25 +136,34 @@ \subsection{Evaluation boards}
   \label{tab:hw:boards}
 \end{table}
 
-\subsubsection{Peripherals}
+\subsection{Peripherals}
 \label{sub:peripherals}
-\todo{This subsection might not be relevant. Do we want a subsection to describe the peripherals of the GG with a little more detail?}
-
-In this project we have only focused on writing software for the Giant Gecko series of microcontrollers.
-We have utilized two different development boards that both features the EFM32GG990 microcontroller,
-thus we have limited implementation of bindings to only support a handful of peripherals that are
-available with this device. A block diagram of the device and its peripherals are shown in
-\autoref{fig:efm32gg990_block_diagram}.
-
-\begin{figure}[b]
-\centering
-\missingfigure{EFM32GG990 block diagram}
-\caption{Some caption}
+
+\autoref{fig:efm32gg990_block_diagram} shows a block diagram of the Giant Gecko.
+The Cortex-M3 and its memory can be found in the upper left corner, it can be seen from the diagram that the \gls{mcu} is connected to all other peripherals over a 32-bit bus.
+This section will shortly introduce the peripherals that are relevant in this project, we will describe what they can be used for and why they are important, but we will not go into detail of how they work.
```

Comment:
add this? `These details can be found in the reference manual for the Giant Gecko \cite{}`